mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -258,15 +300,23 @@ public override void Run (IEnumerable<string> args)
 				v => types.Add (v) },
 			{ "dropns=",
 			  "When processing assembly {ASSEMBLY}, strip off leading namespace {PREFIX}:\n" +
-			  "  e.g. --dropns ASSEMBLY=PREFIX",
-			  v => {
-			    var parts = v.Split ('=');
-			    if (parts.Length != 2) { Console.Error.WriteLine ("Invalid dropns input"); return; }
-			    var assembly = Path.GetFileName (parts [0].Trim ());
-			    var prefix = parts [1].Trim();
-			    droppedAssemblies.Add (assembly);
-			    droppedNamespace = prefix;
-			} },
+			  "  e.g. --dropns=ASSEMBLY=PREFIX",
+				v => {
+					var allDropped = v
+						.Trim ()
+						.Split (' ')
+						.Select (n => n.Split ('='));
+
+					// now process the input
+					foreach(var dropped in allDropped) {
+						var ass = Path.GetFileName (dropped [0].Trim ());
+						var pref = dropped [1].Trim ();
+
+						droppedAssemblies.Add (ass);
+						droppedNamespaces.Add (pref);
+						currentDroppedNamespace = pref;
```

Comment:
...I mean, we're in `Run()`, which is an *instance* method, yet we go off and set a *static* variable? Why?

Yes, the codebase is in bad shape, but we need to clean things like this up, not make it worse!